DMTR-191.tex

```diff
@@ -0,0 +1,546 @@
+\author{G.~Comoretto on behalf of Science Pipelines Team}
+\date{\today}
+
+\setDocRef{DMTR-191}
+
+\setDocAbstract{%
+This brief report describes measurements of interest that were carried out for release v19.0.0 of the Science Pipeline.
+The report for the previous version can be found in \citeds{DMTR-141}.
+}
+
+\newcolumntype{R}[1]{>{\raggedleft\arraybackslash}p{#1}}
+\newcolumntype{L}[1]{>{\raggedright\arraybackslash}p{#1}}
+
+\begin{document}
+
+% Most recent last
+\setDocChangeRecord{
+\addtohist{}{2019-11-26}{First Draft}{G.~Comoretto}
+}
+
+\maketitle
+
+Metrics were measured using \href{https://github.com/lsst/validation_data_hsc}{validation\_data\_hsc}, which consists of 8 HSC engineering images: 2 \emph{r}-band, 4 \emph{i}'-band, and 2 \emph{y}-band. Measurements were made on individual, separately-processed, single-frame images: jointcal was not run. For comparison, we provide the \SRD required ``design'' value of each metric as defined in the Science Requirements Document \citedsp{LPM-17}, and, where available, the target for this release as defined in the Data Management Development Milestone Roadmap \citedsp{LDM-240}. All values were computed using the \texttt{examples/runHscTest.sh} script in the \href{https://github.com/lsst/validate_drp}{validate\_drp} package. For context, the \SRD does not place any constraints on \emph{y}-band for these KPMs.  For the photometric metrics, there are only specifications for \emph{g}, \emph{r}, and \emph{i}'. In the case of the ellipticity correlation metrics, there are specs only for \emph{r} and \emph{i}'. The \emph{y}-band measurements are for interest and historical tracking.
+
+Some KPMs (AF1, AD1) involve thresholds that are different for ``design'', ``minimum'', and ``stretch'' specifications. Thus comparing one of these metrics against a given target number is a two-level process. Both the threshold used in the calculation is dependent on the specifications, and the requirement on the computed number is dependent on the specifications.
+
+The metrics in this report have all been computed relative to the ``design'' thresholds. The values of these KPMs would be different if computed against different thresholds.
+
+Note also that the photometric performance of the pipelines in the \emph{y}-band is an under estimate of expected delivered performance.  For these tests, the \emph{y}-band data was calibrated with \emph{z}-band photometry.  This is due to the lack of a reference catalog containing \emph{y}-band information at this time. We recognize that the bandpass mismatch is certainly not the only source of scatter in the \emph{y}-band photometry. These metric measurements are still worth noting in this report as a historical benchmark to track relative performance.
+
+The \emph{i}-band measurement of TE1 changed somewhat since the previous characterization report.  The new value does not push the TE1 measurement above the ``design'' threshold for that KPM.
```

Comment:
I don't believe this paragraph holds anymore.  Instead, it should mention that virtually all the astrometric metrics have changed for the worse, but that this is completely explained by a change in how we calculate the astrometric quantities as implemented in DM-18751.

Reply:
Would you mind to update the paragraph? 
I am not a scientist, and what you are asking is a scientific comment/narrative. 
You can use the same branch, but if you prefer, open a specific Jira issue or use a user branch.

Reply:
I thought you were taking over responsibility for the content of the report.  If I'm the responsible party, I think I'd like to be on the author list.

Reply:
Of course no problem. 
I think my name can be moved in the last position since I am just editing tables, and actually not adding content.
Also, since the table content is derived from validation_drp Jenkins job, should we include the owner of the package in the review and in the author list also?

Reply:
I propose that @jeffcarlin take responsibility for the characterization metric result as the DM V&V scientist

Reply:
@SimonKrughoff I have added a new paragraph reflecting the changes from DM-18751. Please take a look at the .tex and let me know if you approve.

Reply:
Looks good to me.